migrations/main.sql

```diff
@@ -238,16 +238,16 @@ CREATE TABLE IF NOT EXISTS user_request(
     faculty_id integer NOT NULL,
     university_id integer NOT NULL,
     user_id integer NOT NULL,
-    service_id integer NOT NULL, 
-    date_created timestamp NOT NULL, 
-    status_id integer NOT NULL,
+    service_id integer NOT NULL,
+    date_created timestamp NOT NULL,
+    user_request_status_id integer NOT NULL,
```

Comment:
I don't think it [was supposed](https://euniversity.atlassian.net/browse/EU-92) to rename column `user_request.status_id` to `user_request.user_request_status_id`, old name seems fine. Maybe need clarifications from
@anton-serozhechkin @Kostiantyn-Salnykov

Reply:
@belkka for now it's ok, we will change it in future tickets. We wi;; have only id and name for example